app/services/twitter_service.rb

```diff
@@ -95,7 +95,7 @@ def self.valid_recording(code)
     uri = URI.parse("http://gdata.youtube.com/feeds/api/videos/#{code}")
     Net::HTTP.get(uri)
     video = Yt::Video.new id: code
-    return true if video && video.duration > 2
+    return true if video.live_streaming_details.first.try(:actual_start_time)
```

Comment:
I note I was using `unstarted.live_streaming_detail.actual_start_time` to check the actual start time in the console ...